carapace-server/src/main/resources/conf/cluster.properties

```diff
@@ -23,4 +23,12 @@ db.server.base.dir=dbdata
 # API
 http.admin.enabled=true
 http.admin.port=8001
-http.admin.host=0.0.0.0
+#http.admin.host=0.0.0.0 # default the address of the localhost, achieved by retrieving the name of the host from the system.     
+https.admin.port=-1
+https.admin.sslcertfile=
+https.admin.sslcertfilepassword=
```

Comment:
# admin.advertised.url=http://myserver:8001

we will automatically add /ui#whatever